src/app/services/wallet.service.ts

```diff
@@ -115,6 +116,62 @@ export class WalletService {
     });
   }
 
+  createTransaction(wallet: Wallet, address: string, amount: number): Observable<{ inputs: TransactionInput[], outputs: TransactionOutput[] }> {
+    const addresses = wallet.addresses.map(a => a.address).join(',');
+
+    return this.apiService.getOutputs(addresses)
+      .flatMap((outputs: Output[]) => {
+        const minRequiredOutputs =  this.getMinRequiredOutputs(amount, outputs);
+        const totalCoins = parseInt((minRequiredOutputs.reduce((count, output) => count + output.coins, 0)) + '', 10);
```

Comment:
As the multiplication by 1000000 was removed, it is necessary to remove `parseInt`, or decimal values of the user's outputs will be ignored. Instead filter the value with `Number(([operation]).toFixed(6))`.

Reply:
Fixed.